Packs/Cybereason/Integrations/Cybereason/Cybereason.yml

```diff
@@ -702,12 +775,298 @@ script:
+      if this remediation action is available for that Malop)
+    name: cybereason-kill-prevent-unsuspend
+  - arguments:
+    - description: The unique ID assigned by the Cybereason platform for the Malop
+      name: malopGuid
+      required: true
+    - description: Machine name to prevent a file associated with ransomware
+      name: machine
+      required: true
+    - description: Target ID to prevent a file associated with ransomware
+      name: targetId
+      required: true
+    - description: The complete Cybereason user name string for the user performing
+        the request
+      name: userName
+      required: true
+    - description: Timeout (in seconds) to wait for the remediation response
+      name: timeout
+      defaultValue: "60"
+      auto: PREDEFINED
+      predefined:
+      - "30"
+      - "60"
+    - description: Comment to add to the malop
+      name: comment
+    description: Prevent a file associated with ransomware. (User will get inputs
+      by executing the 'cybereason-available-remediation-actions' command if this
+      remediation action is available for that Malop)
+    name: cybereason-unsuspend-process
+  - arguments:
+    - description: Filter for Fetching Malwares by Malware needsAttention.
+      name: needsAttention
+      auto: PREDEFINED
+      predefined:
+      - "True"
+      - "False"
+    - description: Filter for Fetching Malwares by Malware Type. (Possible filter
+        values for Type are "KnownMalware,UnknownMalware,FilelessMalware,ApplicationControlMalware,RansomwareMalware").
+      name: type
+    - description: Filter for Fetching Malwares by Malware Status. (Possible filter
+        values for Status are "Done,Excluded,Detected,Prevented,Remediated,DeleteOnRestart,Quarantined").
+      name: status
+    - description: Filter for Fetching Malwares by Timestamp. Enter the time (in epoch).
+      name: timestamp
+    - description: Filter for Fetching Malwares by Malware Limit.
+      name: limit
+      required: true
+    description: Malware query with options and values to filter
+    name: cybereason-malware-query
+  - arguments:
+    - description: Sensor ID of a sensor. (Comma separated values supported.)
+      name: sensorID
+      required: true
+    - description: Select a method/type to scan a host.
+      name: scanType
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - "FULL"
+      - "QUICK"
+      - "STOP"
+    description: Start or stop a full or quick scan for a host.
+    name: cybereason-start-host-scan
+  - arguments:
+    - default: true
+      description: The batch ID obtained after initiating the scan.
+      name: batchID
+      required: true
+    description: Get the results for host scanning.
+    name: cybereason-fetch-scan-status
+  - arguments:
+    - default: false
+      description: The hostname of the machine.
+      isArray: false
+      name: machineName
+      required: false
```

Comment:
Providing explanation of what happens when no id is specified here would be appriciated by customers.

Reply:
We are quite unsure about which id you are referring here, as the command just accepts the machine name as input

Reply:
I was referring to the machine name is an identifier, but I saw you handled the case of empty input in the command code, so its ok by me.